README.md

```diff
@@ -94,11 +154,11 @@ Vue Storefront solves a set of key business challenges from the world of the sho
 
 ## Contributing
 
-If you like the ideas behind Vue Storefront and want to become a contributor - join our [Discord server](https://discord.vuestorefront.io), check the [list of the active issues](https://github.com/vuestorefront/vue-storefront/issues) or contact us directly via contributors@vuestorefront.io.
+If you like the ideas behind Vue Storefront and want to become a contributor - join our [Discord server](https://discord.vuestorefront.io), check the [list of the active issues](https://github.com/vuestorefront/vue-storefront/issues) or contact us directly via contributors(at))vuestorefront(dot)io.
```

Comment:
Why do we need these "(at)" and "(dot)" instead of "@" and "."?

Reply:
To prevent more crawling on the email